### PR TITLE
Document partial build functionality

### PR DESCRIPTION
**Pull request by benjamingr**
Adds a description of how to make a partial build of Ramda, closes https://github.com/ramda/ramda/issues/806

I will create a separate PR for the docs repo as it's in a different repo. Note I've also included the /docs fix here.

### CODE REVIEW

**Comment by benjamingr** [comment]
Hey, in first look it looks like the index file in the docs repo is generated from the README.md file here - is this the case or should I submit a separate PR for the docs repo?

File: README.md [hunk 1]
```diff
@@ -120,10 +120,18 @@ Or you can inject ramda into virtually any unsuspecting web site using this book
...
 ```
 
+### Partial Builds
+
+It is possible to build Ramda with a subset of the functionality to reduce its file size. Ramda's build system supports this with command line flags. For example if you're using `R.compose`, `R.reduce` and `R.filter` you can create a partial build with:
+
+    ./scripts/build -- src/curry.js src/compose.js src/reduce.js src/map.js src/filter.js > dist/ramda.custom.js
```
Comment (buzzdecafe) [inline README.md]: i'd say that the functions in the description should match the functions in the example. So the preferable way to go is probably to remove `src/curry.js` and `src/map.js` from the example

Comment (benjamingr) [inline README.md]: Roger. Editing.

**Comment by buzzdecafe** [comment]
> Hey, in first look it looks like the index file in the docs repo is generated from the README.md file here - is this the case or should I submit a separate PR for the docs repo?

no need to pr vs. ramda.github.io

**Comment by benjamingr** [comment]
@buzzdecafe thanks. In that case the PR is done - you can merge it if you'd like.

Great work on the library, having something powerful like this in JS is really refreshing, as someone who's never seen the point of underscore given the native methods this has shown me a new light :) Asked that question and started this PR since we're actually deploying an internal app using Ramda. Thanks again for saving me time with the library.

**Comment by buzzdecafe** [comment]
thanks for the kind words. can you please squash to one commit? then looks good to merge

**Comment by benjamingr** [comment]
Done

**Comment by buzzdecafe** [comment]
thanks!

**Pull request merged**